README.md

```diff
@@ -26,12 +26,12 @@ Increase test [performance examples](https://github.com/jdi-testing/jdi-light/tr
 JDI – is the test Framework for UI test automation that helps to makes your tests fast and sustainable and provide obvious and predictable test run result
 
 ## Highlights
--   Has simple integration with any Selenium based project that allows to increase projects stability, add user actions logs and enrich standard WebElements capabilities with new features in minutes (TBD add Guide)
--   Extends Page Objects pattern with powerful UI Elements: Buttons, TextFields, Forms, Tables etc ( TBD add good links to examples)
--   Write stable and predictable tests that fail only because of changing business logic or layout. No more waits, thread sleep and other flaky staff
--   Increase your tests execution speed in general and especially for complex cases like search in big tables, dropdowns with hundreds elements or enter text with thousand lines
--   Get detailed logs and good reports of all user actions with no effort
--   Easy integration with all modern Automation tools: CI (Jenkins, TC etc.), Logging (Log4j or any other slf4j based logs), Reporting (Allure or Report Portal); Browser/Devices farms (Selenium Grid, Browser Stack, Selenoid etc.), Test Runners (TestNG, JUnit) etc.
+- Has simple integration with any Selenium based project that allows to increase projects stability, add user actions logs and enrich standard WebElements capabilities with new features in minutes (TBD add Guide)
+- Extends Page Objects pattern with powerful UI Elements: Buttons, TextFields, Forms, Tables etc ( TBD add good links to examples)
+- Write stable and predictable tests that fail only because of changing business logic or layout. No more waits, thread sleep and other flaky staff
+- Increase your tests execution speed in general and especially for complex cases like search in big tables, dropdowns with hundreds elements or enter text with thousand lines
+- Get detailed logs and good reports of all user actions with no effort
+- Easy integration with all modern Automation tools: CI (Jenkins, TC etc.), Logging (Log4j or any other slf4j based logs), Reporting (Allure or Report Portal); Browser/Devices farms (Selenium Grid, Browser Stack, Selenoid etc.), Test Runners (TestNG, JUnit) etc.
```

Comment:
It doesn't connect to issue 990. And it causes a lot of new Codacy issues:
https://app.codacy.com/manual/jdi-testing/jdi-light/pullRequest?prid=4361169